src/webrtc/groupCall.ts

```diff
@@ -472,6 +473,12 @@ export class GroupCall extends TypedEventEmitter<
 
         for (const call of this.calls) {
             call.localUsermediaFeed.setAudioVideoMuted(muted, null);
+
+            // Disable voice activity detection when user has muted their microphone
+            // (This can't happen when push-to-talk is enabled)
+            if (!this.isPtt) {
+                call.localUsermediaFeed.VADEnabled = !muted;
+            }
```

Comment:
So I don't think we should be doing anything here - I assume we would only want VAD enabled in the `GroupCall`'s `localUsermediaFeed` and then use that to inform if the other clones should be muted

Reply:
I.e. this should be under `if (this.localCallFeed) {...`